src/main/routes/confirmDeleteItem.ts

```diff
@@ -0,0 +1,28 @@
+const router = require("../routes/home");
+
+/* GET */
+router.get("/deleteitem", (req, res, next) => {
+  const responseContent: { [k: string]: any } = {};
+  responseContent.response = req.session.response;
+  responseContent.itemToDelete = req.query.item;
+
+  if (req.query.item === "user") {
+    responseContent.idamId = req.query.idamId;
+    responseContent.warning = `Are you sure you would like to delete user ${req.query.idamId}?`;
```

Comment:
Just an observation, many places use a double quote for a string and this line uses ``.

Reply:
That's deliberate, given that the ``` is used for template strings, but this one contains no variables and is a regular string.

Reply:
Thank you for reminding this old man

Reply:
:) :+1: